pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_space_foi.py

```diff
@@ -69,6 +70,35 @@ def test_cell_inf(self):
         self.assertIsInstance(result, float)
         self.assertTrue(result >= 0)
 
+    @patch('pyEpiabm.property.SpatialInfection.space_susc')
+    @patch('pyEpiabm.property.SpatialInfection.space_inf')
+    @patch('pyEpiabm.core.Parameters.instance')
+    def test_carehome_scaling(self, mock_params, mock_inf, mock_susc):
+        mock_inf.return_value = 1
+        mock_susc.return_value = 1
```

Comment:
Is it possible to check these have been called in the way you expect (or at least the number of times you expect?